src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -37,9 +37,14 @@ const HeatmapPlot = (props) => {
   const properties = useSelector((state) => state.cellSets.properties);
   const hidden = useSelector((state) => state.cellSets.hidden);
 
+  const selectedTracks = useSelector((state) => state.cellInfo.selectedTracks);
+  const groupedTrack = useSelector((state) => state.cellInfo.groupedTrack);
```

Comment:
Have you thought about whether this approach still keeps our redux store extensible -- for instance, will values for `selectedTracks` and `groupedTrack` apply to all components similarly to how the `cellId` in the `cellInfo` redux store applies to everything? Or would values of `selectedTracks` and `groupedTrack` be different for different views of the dataset?
Just wondering whether `cellInfo` is the right place in the store to be placing these.